pkg/controller/core/workload_controller_test.go

```diff
@@ -182,103 +182,91 @@ func TestAdmittedNotReadyWorkload(t *testing.T) {
 func TestSyncCheckConditions(t *testing.T) {
 	now := metav1.NewTime(time.Now())
 	cases := map[string]struct {
-		conds                []metav1.Condition
+		conds                []kueue.AdmissionCheckState
```

Comment:
```suggestion
		states                []kueue.AdmissionCheckState
```

and `wantStates` below